markdown-it-decksetter.js

```diff
@@ -38,13 +38,11 @@ module.exports = function containerPlugin (md, name, options) {
     var max = state.eMarks[startLine] // the index of the last character of a line
 
     // if this is the first iteration and the marker is not found, this is the first slide
-    if (markerChar !== state.src.charCodeAt(start)) {
+    if (start === 0) {
```

Comment:
Made this change because `isTop` is always going to be `true` when `0`, so we won't need to look at the `markerChar` and `state.src` char's to figure that out.